app/containers/MessageActions/Header.tsx

```diff
@@ -93,9 +92,9 @@ const HeaderItem = ({ item, onReaction, server, theme }: THeaderItem) => {
 const HeaderFooter = ({ onReaction, theme }: THeaderFooter) => (
 	<Button
 		testID='add-reaction'
-		onPress={onReaction}
+		// this is not the correct, but but we have a lot of "buttons" and a don't know what the font of truth
+		onPress={onReaction as () => void}
```

Comment:
I don't think it's a matter of source of truth.
RectButton uses 
```tsx
onPress?: (pointerInside: boolean) => void;
```
while you're sending 
```tsx
({ emoji }: { emoji: TItem }) => void
```

Since Touch is a generic component, I'd override the type with `Function`.

Also we need to reevaluate [this](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/containers/ActionSheet/Button.ts#L7), since it's taken from a lib we're not using anymore.